### PR TITLE
Weapon sharpness and edge adjustments

### PR DESCRIPTION
**Pull request by Yoshax**
Makes several weapons sharp, but not have an edge.
- Forks
- Harpoons
- Spears

Also makes screwdrivers sharp.

### CODE REVIEW

File: polaris.dme [hunk 1]
```diff
@@ -1897,9 +1897,4 @@
 #include "code\ZAS\Zone.dm"
 #include "interface\interface.dm"
 #include "interface\skin.dmf"
-#include "maps\polaris-1.dmm"
-#include "maps\polaris-2.dmm"
-#include "maps\polaris-3.dmm"
-#include "maps\polaris-4.dmm"
-#include "maps\polaris-5.dmm"
```
Comment (PsiOmegaDelta) [inline polaris.dme]: Removing the map?

Comment (Yoshax) [inline polaris.dme]: Oops.

**Comment by Datraen** [comment]
:+1:

**Comment by Spacemanspark** [comment]
You're losing your edge.

**Pull request merged**